<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InformeParcelasDockWidgetBase</class>
 <widget class="QDockWidget" name="InformeParcelasDockWidgetBase">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>326</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Informe Parcelas</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" rowspan="3">
     <widget class="QGroupBox" name="salidasInformesBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>175</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="title">
       <string>SALIDAS INFORME</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <widget class="QLineEdit" name="csvcFileLineEdit">
       <property name="geometry">
        <rect>
         <x>98</x>
         <y>51</y>
         <width>591</width>
         <height>20</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="csvcFilePushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>82</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Archivo CSV</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="kmlFileLineEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>591</width>
         <height>20</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="kmlFilePushButton">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>109</y>
         <width>82</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Archivo KML</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="planoFieldCheckBox">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Crear Plano (pdf)</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="csviFilePushButton">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>79</y>
         <width>82</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Archivo Prop.</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="csviFileLineEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>591</width>
         <height>20</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="listadoFieldCheckBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Crear Listado (pdf)</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="terminarPushButton">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>140</y>
         <width>82</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Terminar</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="seleccionBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="title">
       <string>SELECCION</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QPushButton" name="selec_ParcelaPushButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>80</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Parcela</string>
       </property>
      </widget>
      <widget class="QPushButton" name="selec_PropietarioPushButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>80</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Propietario</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="n_parcelaLineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>80</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="n_propietarioLineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>50</y>
         <width>80</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
